### DISCUSSION TITLE
Block programming - international translations?

### DISCUSSION THREAD

**Discussion by biasedlogic** [pybricks/support#1456] [Ideas]
Hi, if I see it correctly, PyBricks is right now offered only in English. While, considering Python is also English-based, like most textual programming languages, it does not disturb most user base there, the Block Coding part could really profit from being translated.
Is this planned? (or maybe it already has, and I'm just too dumb to see the language switch...?)
If it is on the roadmap: I can help with German and Polish :-)

**Comment by dlech** [comment]
The official feature request is at https://github.com/pybricks/support/issues/1345, so you can give that issue a 👍 and subscribe to it for updates.

**Comment by biasedlogic** [comment]
Thanks, I've searched discussions, but didn't find that issue. I'm not sure though if that one covers the block programming? Anyway, I've subscribed there and offered my help for German and Polish.
